DESCRIPTION

```diff
@@ -35,8 +35,8 @@ Suggests:
   miniUI, 
   shiny, 
   rmarkdown
-Depends: ggplot2 (>= 3.4.0)
-RoxygenNote: 7.2.3
+Depends: ggplot2 (>= 3.5.0)
```

Comment:
@kyleam I upgraded ggplot2 but still wasn't able build the vignettes. The error was

```r
Quitting from lines 45-51 [unnamed-chunk-4] (list_plots.Rmd)
Error: processing vignette 'list_plots.Rmd' failed with diagnostics:
'==' only defined for equally-sized data frames
```

and I think this is coming from patchwork?  So I updated patchwork and seems like it's building again. So can we (should we) require patchwork >= 1.2.0 as well?

Reply:
Good catch.  I was testing with only the latest patchwork.

Looks like patchwork 1.2.0 fixed this compatibility issue with [0033fc0](https://github.com/thomasp85/patchwork/commit/0033fc0ec06d3da682acf344eff698edc927ac59) (Rework `add_guides` for arbitrary guide positions, 2024-01-04).

> So can we (should we) require patchwork >= 1.2.0 as well?

I think that makes sense.  Done with 86bd666.